tests/vmi_tpm_test.go

```diff
@@ -0,0 +1,82 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2022 Red Hat, Inc.
+ *
+ */
+
+package tests_test
+
+import (
+	expect "github.com/google/goexpect"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"kubevirt.io/client-go/kubecli"
+
+	v1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/kubevirt/tests"
+	"kubevirt.io/kubevirt/tests/console"
+)
+
+var _ = Describe("[sig-compute]vTPM", func() {
+	var virtClient kubecli.KubevirtClient
+	var err error
+
+	BeforeEach(func() {
+		virtClient, err = kubecli.GetKubevirtClient()
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	Context("with TPM VMI option enabled", func() {
+		It("should expose a functional emulated TPM which persists across migrations", func() {
+			By("Creating a VMI with TPM enabled")
+			vmi := tests.NewRandomFedoraVMI()
```

Comment:
nit: Can we use `libvmi`?

Reply:
So, the code for this function actually uses `libvmi`, and also adds all the necessary stuff. Using `libvmi` directly I would effectively rewrite that function. Better to use it IMO.

Reply:
I wonder what is special about the function and how many usages it has but that's for another PR.